scripts/init_db.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -ex # hanldling errors and Debuging purpose 
+# Declare environment variables
+
+DB_USER="${POSTGRES_USER:=postgres}"
+DB_PASSWORD="${POSTGRES_PASSWORD:=password}"
+DB_NAME="${POSTGRES_DB:=newsletter}"
+DB_PORT="${POSTGRES_PORT:=5432}"
+DB_HOST="${POSTGRES_HOST:=localhost}"
+SKIP_DOCKER=${SKIP_DOCKER:=false}
+DATABASE_URL="postgres://$DB_USER:$DB_PASSWORD@localhost:5432/$DB_DATABASE"
+
+# Check if Docker is installed
+if [ "$SKIP_DOCKER" = "false" ]; then
```

Comment:
SKIP_DOCKER is a boolean not a string. It makes no sense to have it as a string as we need to only use it as a check.
```suggestion
if [[ -z "${SKIP_DOCKER}" ]]; then
```